doc/source/overview/saving.md

```diff
@@ -143,26 +143,34 @@ the scikit-learn API.
 ```
 ````
 
-## Saving state
+## Online detectors
 
-[Online drift detectors](../cd/methods.md#online) are stateful, with their state updated upon each `predict` call. 
-When saving an online detector, the `save_state` option controls whether to include the detector's state:
+[Online drift detectors](../cd/methods.md#online) are stateful, with their state updated each timestep `t` (each time
+`.predict()` or `.state()` is called). {func}`~alibi_detect.saving.save_detector` will save the state of online 
```

Comment:
Do you mean `score()` instead of `state()`? On that note, do we even document the usage/use cases of `score()`? If not, perhaps should leave it out.

Reply:
Good spot thanks. Also fair point about not really documenting it. I'll remove.

Reply:
Removed `.state()` in [d888276](https://github.com/SeldonIO/alibi-detect/pull/604/commits/d88827651e6078e065140ab6c601e4cd0199e224)